src/Nancy/GlobalizationConfiguration.cs

```diff
@@ -15,15 +15,22 @@ public class GlobalizationConfiguration
         /// <summary>
         /// Initializes a new instance of the <see cref="GlobalizationConfiguration"/> class
         /// </summary>
-        /// <param name="supportedCultureNames"></param>
-        public GlobalizationConfiguration(IEnumerable<string> supportedCultureNames)
+        /// <param name="supportedCultureNames">An array of supported cultures</param>
+        /// <param name="defaultCulture">The default culture of the application</param>

```

Comment:
That's exactly what is done to determine the Context.CurrentCulture. You wouldn't be using the configuration default culture anyway I hope, you should be using the Context.CurrentCulture